redis/src/main/scala/zio/redis/options/Connection.scala

```diff
@@ -27,38 +33,98 @@ trait Connection {
 
   sealed case class ClientEvents(readable: Boolean = false, writable: Boolean = false)
 
-  sealed trait ClientFlag
+  sealed trait ClientFlag {
+    private[redis] def stringify: String
```

Comment:
Perhaps implement it here, less "noise" later :).

Reply:
updated this in the last commit (merged with master)